bot/resources/evergreen/py_topics.yaml

```
@@ -1,25 +1,29 @@
-# Conversation starters for python-related channels.
+# Conversation starters for Python-related channels.
 
 # python-general
-267624335836053506:
+704362672644423840:
```

This isn't the correct ID for `#python-general`.
```suggestion
267624335836053506:
```